languages/de-DE/Buttons.multids

```diff
@@ -30,12 +30,22 @@ ExportTiddler/Caption: Exportieren
 ExportTiddler/Hint: Diesen Tiddler Exportieren
 ExportTiddlers/Caption: Mehrere Exportieren
 ExportTiddlers/Hint: Mehrere Tiddler Exportieren
+Fold/Caption: Ausblenden Textbereich
```

Comment:
yea, but drop down lists should have the action info as the very first info. So users don't need to read:

 - Textbereich ausblenden
 - Textbereich einblenden
 - Textbereich aller einblenden
 - Textbereich aller ausblenden

The visual difference between the different actions is very low. So it's error prone.